draft-rescorla-tls-subcerts.md

```diff
@@ -300,37 +342,36 @@ TLS stack, which has the advantage of avoiding changes to security-critical and
 often delicate PKI code (though of course moves that complexity to the TLS
 stack).
 
-[[ OPEN ISSUE: If we use this syntax, how would you represent non-signature
-keys? ]]
-
-As in the above case, there would be a need for a special marker in the
-master certificate that declares that the key pair can be used to issue
-sub-certificates.
+By only allowing this signature to use TLS 1.3 signature schemes, we prevent cross
+protocol attacks such as those noted in [Jager et al.]. We could also avoid
+complicating the signature scheme selection logic by requiring the publicKey of
+the delegated credential to be the same size and format as the key in the leaf
+certificate.
```

Comment:
would be nice to not have this restriction to speed up adoption of things like new curves without requiring CA intervention.